__tests__/apps.test.js

```diff
@@ -113,6 +113,28 @@ describe("app", () => {
                 
             });
         })
+        describe('POST /api/articles/:article_id/comments', () => {
```

Comment:
Lets do another test here that checks the post ignores unnecessary properties
For example if we posted - { username: "icellusedkars", body: "this is a new comment", fruit:"apple"} - it would give a 201 and would post successfully but would not include fruit:apple as the data isn't inserted :)